src/hotkey/queue.c

```diff
@@ -0,0 +1,190 @@
+#include "queue.h"
```

Comment:
queue a bit too generic given we have a cc_queue in ccommon

Can we call it something more context-aware such as key_window or something (not thinking very hard here right now)